zaza/openstack/charm_tests/designate/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Canonical Ltd.
```

Comment:
I'm curious as to why this base class definition is in `__init__.py` and not just before the only usage of the class? Why not put it in the `tests.py` file?

Reply:
i think i was following the keystone tests as a template.  Maybe it's the one off instead of the standard.  I'll move it.